.gitignore

```diff
@@ -1,5 +1,6 @@
 .gradle
-/build/
+**/**/build/
+**/**/bin/
```

Comment:
`**/**/build/` can be simplified to just `**/build/`.  This will match all the build directories wherever they are - see https://git-scm.com/docs/gitignore#_pattern_format.  The same applies to `**/**/bin` although it might be an idea to change it to `**/bin/**` so that it excludes all directories under bin but not the `run-in-docker.sh` file in the `/bin` directory.

Reply:
**/build/
**/bin/

Changed to above now